.github/workflows/main.yml

```diff
@@ -1,13 +1,6 @@
 name: Build
 
 on:
-  push:
```

Comment:
This is to not build on a push and just need to be reintegrated, correct?

Reply:
Yes that's correct. For some reason the PR merged to dev to stop building on push didn't get merged into the feature branch using `git merge`. Couldn't figure out why so I just re-deleted